src/components/SwapWidget/subcomponents/InfoSection/InfoSection.styles.tsx

```diff
@@ -8,9 +8,14 @@ import IconButton from "../../../IconButton/IconButton";
 import { InfoHeading, InfoSubHeading } from "../../../Typography/Typography";
 import { StyledParagraph } from "../../../Typography/Typography.styles";
 
-export const StyledInfoHeading = styled(InfoHeading)`
+type StyledInfoHeadingProps = {
+  serverUrl?: string | null;
```

Comment:
what

Reply:
this was to add a margin above `"Selected as {{serverUrl}} custom server"`. If that message doesn't render, the margin is fine. For some reason margin is needed when that message is there

![image](https://github.com/airswap/airswap-web/assets/90465420/43a3a9d5-1652-4c7e-88ef-f1b0763209f5)

Reply:
1. Using a variable is unnecessary.
2. The margin is wacky because there's margin around the clear custom server

Reply:
Yeah it was unnecessary. I removed it